### DISCUSSION TITLE
How to make controls look like "native" Windows controls

### DISCUSSION THREAD

**Discussion by tms320** [AvaloniaUI/Avalonia#11717] [General]
Hello!
I am new with Avalonia framework.
I have created Avalonia desktop project. But the style of controls don't look like "native" Windows controls:
![image](https://github.com/AvaloniaUI/Avalonia/assets/24802646/f4f9f5b1-70bd-4cc9-9c90-61fc04b7393e)
For example, I don't like large blank space around ListBox items.
I want my Avalonia controls look like this (WPF application):
![image](https://github.com/AvaloniaUI/Avalonia/assets/24802646/4d0889d4-899c-4ef4-a7ee-36fecf22229c)
How can I implement this "native" style in Avalonia project?
PS: here is my AvaloniaTest.csproj created by Visual Studio:
![image](https://github.com/AvaloniaUI/Avalonia/assets/24802646/91195f1d-7c39-4769-9714-b50b0f5e7239)

**Comment by maxkatz6** [comment]
> How to make controls look like "native" Windows controls

What you are showing is not a native Windows control. It's a ~win7 style.
Avalonia is closer to native Win10 control styles. With third party [FluentAvalonia](https://github.com/amwx/FluentAvalonia) which makes it closer to Win11.

There is another third party for [avalonia aero theme](https://github.com/Splitwirez/avalonia-aero-theme). Check it out if you want old windows styles.

**Reply by maxkatz6** [reply]
You also can style default controls, change padding on ListBoxItem, etc...

**Reply by timunie** [reply]
Note: Avalonia is made to look the same on each platform and not made to match platform style. So if you style the App to look like Windows 7 for example, it will also look like that on Mac or Linux.
